net.js

```diff
@@ -68,6 +68,16 @@ SSB.getThread = function(msgId, cb) {
   })
 }
 
+SSB.connected = function(cb) {
```

Comment:
This change is unrelated right?

Reply:
(Looks like my reply may have gone to the wrong place.  Trying again.)

No, that was needed to get the "Get msg" buttons working, because the ssb-msg template's getOOO calls SSB.getOOO, which tries to call SSB.connected. That fails because SSB.connected didn't exist, so I had to build an SSB.connected function based on inferring from SSB.getOOO what it was supposed to do.

Reply:
Right, that was a leftover. I'll fix that a little bit differently.